scripts/pod_lib_lint.rb

```diff
@@ -29,6 +29,12 @@ def usage()
   podspec is the podspec to lint
 
   options can be any options for pod spec lint
+
+  script options:
+    --ignore-local-specs: list of podspecs that should not be added to
```

Comment:
`pod lib lint` calls this `--external-podspecs`. Make this consistent with the underlying flag?

Indeed, why not just just pass this through?

Reply:
It is still not exactly clear to my what `--external-podspecs` option means, but if I pass it, then the specified spec file is used to lint. Here are the current script results run on [mm/iid-with-fis](https://github.com/firebase/firebase-ios-sdk/tree/mm/iid-with-fis):
```
./scripts/pod_lib_lint.rb FirebaseInstallations.podspec                                                                                                               
bundle exec pod lib lint --sources=https://cdn.cocoapods.org/ --include-podspecs={FirebaseCore.podspec,GoogleUtilities.podspec,FirebaseCoreDiagnosticsInterop.podspec,FirebaseCoreDiagnostics.podspec,GoogleDataTransportCCTSupport.podspec,GoogleDataTransport.podspec,FirebaseInstanceID.podspec} FirebaseInstallations.podspec

 -> FirebaseInstallations (0.1.0)
    - WARN  | url: The URL (https://twitter.com/Firebase) is not reachable.
    - ERROR | [iOS] unknown: Encountered an unknown error (CocoaPods could not find compatible versions for pod "FirebaseInstanceID":
  In Podfile:
    FirebaseInstallations/unit (from `/Users/mmaksym/Projects/firebase-ios-sdk`) was resolved to 0.1.0, which depends on
      FirebaseInstanceID (~> 4.2.0)

    FirebaseInstanceID (from `.`)) during validation.

[!] FirebaseInstallations did not pass validation, due to 1 error and 1 warning.
You can use the `--no-clean` option to inspect any issue.
```

```
./scripts/pod_lib_lint.rb FirebaseInstallations.podspec --platforms=ios --external-podspecs=FirebaseInstanceID.podspec --ignore-local-specs=FirebaseInstanceID.podspec
bundle exec pod lib lint --sources=https://cdn.cocoapods.org/ --include-podspecs={FirebaseCore.podspec,GoogleUtilities.podspec,FirebaseCoreDiagnosticsInterop.podspec,FirebaseCoreDiagnostics.podspec,GoogleDataTransportCCTSupport.podspec,GoogleDataTransport.podspec} FirebaseInstallations.podspec --platforms=ios --external-podspecs=FirebaseInstanceID.podspec

 -> FirebaseInstallations (0.1.0)
    - ERROR | [iOS] unknown: Encountered an unknown error (CocoaPods could not find compatible versions for pod "FirebaseInstanceID":
  In Podfile:
    FirebaseInstallations/unit (from `/Users/mmaksym/Projects/firebase-ios-sdk`) was resolved to 0.1.0, which depends on
      FirebaseInstanceID (~> 4.2.0)

    FirebaseInstanceID (from `FirebaseInstanceID.podspec`)) during validation.

[!] FirebaseInstallations did not pass validation, due to 1 error.
You can use the `--no-clean` option to inspect any issue.
```

```
./scripts/pod_lib_lint.rb FirebaseInstallations.podspec --platforms=ios  --ignore-local-specs=FirebaseInstanceID.podspec
bundle exec pod lib lint --sources=https://cdn.cocoapods.org/ --include-podspecs={FirebaseCore.podspec,GoogleUtilities.podspec,FirebaseCoreDiagnosticsInterop.podspec,FirebaseCoreDiagnostics.podspec,GoogleDataTransportCCTSupport.podspec,GoogleDataTransport.podspec} FirebaseInstallations.podspec --platforms=ios

 -> FirebaseInstallations
...

Success
```

Reply:
`-external-podspecs` is used to specify a path to a podspec to use instead of what is found locally or via `--sources`.  One approach would be to checkout the `InstanceID-4.2.2` tag and use the podspec from that - but that doesn't seem worth it.

Reply:
I see. Fair enough that this isn't `--external-podspecs`.

Consider at least naming this consistently with `pod lib lint`'s options. It uses `--include-podspecs` rather than `--include-specs`, so that would make your new argument `--ignore-local-podspecs`.

Reply:
Updated to `--ignore-local-podspecs`